src/components/Icons/Clickable.tsx

```diff
@@ -25,17 +25,17 @@ const Clickable = (props: IClickableProps | any) => {
 		setHovered(false);
 	};
 
-	const clickedStyles = clicked && 'border border-3 p-1 flex-shrink-1';
-	// TODO need to get centering correct so that there is not jittering
-	const hoverStyles = false && hovered && 'border';
+	// Clicked-style is prioritised, then hovered-style, otherwise fallback to default style.
+	const clickedOrHoveredStyle = clicked ? 'bg-info' : hovered ? 'bg-light' : 'bg-body'
```

Comment:
I think the outline is a bit too thick. Can you make it a bit thinner and also add 
```js
style: {
  transition: '0.2s'
}
``` so that it doesn't change abruptly? Maybe even 0.1s, whatever feels more pleasant

Reply:
Transition is added.

I'm not happy about the smaller padding. Primary, because we'd replace bootstrap-css with custom CSS. But also because with custom css, I don't get the perk icon perfectly centered. I did however add the Bootstrap class `rounded-1`, which makes the "border" (It's really just background color) a little easier on the eye.

Bootstrap `p-1` class:
![image](https://user-images.githubusercontent.com/2971735/173189197-e909d77e-9e81-4d24-a050-10a37e760ed9.png)

Custom style `padding: 0.175rem`
![image](https://user-images.githubusercontent.com/2971735/173189223-36806f0e-9f8b-4fa3-b270-e8dc02ddd2ec.png)

Reply:
I like the rounded with p-1! 
Let's keep that